webserver/views/user.py

```diff
@@ -29,18 +29,20 @@ def lastfmscraper(user_id):
 
 @user_bp.route("/<user_id>")
 def profile(user_id):
-    cassandra = webserver.create_cassandra()
+    # Which database to use to showing user listens.
+    #db_conn = webserver.create_cassandra()
+    db_conn = webserver.create_postgres()
```

Comment:
I am still not sure if i must remove the cassandra support completely. In that i may need to do some code refactoring, initially i tried to make it compatible with both. I think its better to just remove Cassandra code.

@mayhem fine?